dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -28,6 +28,22 @@ If the titles of any of the below analyzers change, ensure [HelpLinkUrlFactory](
 
 To run VS experimentally set the `.Vsix` project as the startup project, which launches visual studio with the analysers installed. Depending on your use case create or open an existing project. Code that violates any analysers in this project should then be flagged.
 
+## Updating the workspace to support a new C# version
```

Comment:
Can you tweak the wording of this heading please? Something like `Updating the analyzers to support a new C# version`.

Reply:
Absolutely. Done :)